### ISSUE TITLE
Remote not work on Google tv

### ISSUE THREAD

**Issue by Monuk2744** [Zenda-Cross/vega-app#158]
### Describe your suggested feature

The remote is not working, I just have to slide it up and down or back 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Zenda-Cross** [comment]
Yes, I am aware of this issue: the app does not support tvOS, and I am unsure if it will be fixed in the near future.

**Comment by rajbhanushali97** [comment]
> ### Describe your suggested feature
> The remote is not working, I just have to slide it up and down or back
> 
> ### Acknowledgements
> * [x]  I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

how u managed to get it installed on smart TV ?
